packages/itwinui-react/src/core/Table/Table.test.tsx

```diff
@@ -37,6 +37,7 @@ import {
   SelectionColumn,
   ExpanderColumn,
 } from './columns/index.js';
+import exp from 'node:constants';
```

Comment:
?

```suggestion
```

Reply:
This might have been an auto-import as I typed "expect". I have removed this.